flaxandteal/coral-arches
# Feat/2295 excavation dashboard frontend

## Description
Componetised the dashboard card and added a state to allow switching of layouts and variables depending on the user and the dashboard

## Testing
- Need to register the card component
- Create a user
- Import the Excavation Team Permissions
- Add the user to the Excavation Admin Group
- Create a new excavation licence
- Fill in the fields
- Check that the task is showing with the correct fields
- Change sort by options, check they are sorting correctly
- Check the open button takes you to the correct workflow
- Add a new report - check the status updates
- Repeat the tests for both a Cur D user and Cur E user added to the appropriate groups
